source/common/res/features/import-notification/main.js

```diff
@@ -22,4 +23,3 @@
     setTimeout(poll, 250);
   }
 }());
-
```

Comment:
@joshmadewell, we've actually already got it turned on here: https://github.com/toolkit-for-ynab/toolkit-for-ynab/blob/master/.editorconfig#L7

@dbaldon, are you using EditorConfig for your editor? If not, you can install the extension for most editors and it'll help you format as per the eslint rules to save you some effort.